content/en/docs/upgrading/upgrade.md

```diff
@@ -137,7 +137,7 @@ Alternatively you can follow these steps to change the configuration before appl
 ## Upgrades from earlier versions of Kubeflow
 
 For earlier versions, Kubeflow makes no promises of backwards compatibility or 
-upgradeability. Nonetheless, here are some instructions for updating your deployment:
+upgradability. Nonetheless, here are some instructions for updating your deployment:
```

Comment:
Can you also check the following if you have time @kelvins :

---

Line 24:

- `Make sure` <- `Ensure`

```diff
- * Ensure that your Kubeflow namespace is annotated with the
+ * Make sure that your Kubeflow namespace is annotated with the
```

---

Line 30:

- For a11y: maybe maybe we can use `Your output should be ...` instead of `You should see...`

```diff
-    You should see something like:
-    Your output should be similar to this:
```

---

Line 183:

- For a11y, as before

```diff
- See the [Kubeflow Pipelines upgrade guide](/docs/pipelines/upgrade/).
+ Refer to the [Kubeflow Pipelines upgrade guide](/docs/pipelines/upgrade/).
```

Reply:
Great suggestions. Done :heavy_check_mark: